libhttpserver/server.go

```diff
@@ -136,40 +158,65 @@ func overrideMimeType(ext, mimeType string) (newExt, newMimeType string) {
 	return ext, mimeType
```

Comment:
Why do we have a custom mechanism here for `javascript` and `html`, but below we also have `additionalMimeTypes`? Why not add those extensions to the `additionalMimeTypes` list?

Reply:
`additionalMimeTypes` is based on exact ext/mimeType paris, while the overrider allows other ways of matching. For example, there could be some other ext that matches to `text/html`. The overrider will be able to catch them but `additionalMimeTypes` can't. The reason why `additionalMimeTypes` exists is that on some platform (where cat /etc/apache2/mime.types is populated), stuff like `.pub` are application types, and cause the webview to prompt a download dialog. So we explicitly patch them.

Reply:
OK. Do we have that documented in the code somewhere?

Reply:
I had some comments noting that we should update both list together. But will add more about this in `libmime`.